README.md

```diff
@@ -5,7 +5,7 @@ Leanplum SDK integration for the Segment iOS SDK.
 To install the Leanplum Segment integration, simply add this line to your
 [CocoaPods](https://cocoapods.org) Podfile:
 ```ruby
-pod 'LeanplumSegment', '~> {{LP_SEG_VERSION}}'
+pod 'LeanplumSegment', '~> 1.1.1-SNAPSHOT'
```

Comment:
yes, the version will be updated in release branch via `./bump_version.sh OLD_VERSION NEW_VERSION`